### PR TITLE
fix datatime error

### PR DESCRIPTION
**Pull request by Sen-Zhang**
fix for https://github.com/rails/rails/issues/24195

### CODE REVIEW

**Comment by olimart** [comment]
Hi @arthurnn, any idea why it's not been merged yet?
Also encountering the same issue with rails 5.0.0.1

**Comment by arthurnn** [comment]
@Sen-Zhang thanks for the PR.
Can you add a changelog for this before I merge it?

Thanks

**Comment by Sen-Zhang** [comment]
@arthurnn Sure, I will

**Comment by Sen-Zhang** [comment]
@arthurnn I have added the changelog, please take a review. Thanks

File: guides/CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
+## Rails 5.1.0 (Nov 01, 2016) ##
+
+*   Fix an Active Record DateTime field NoMethodError caused by incomplete 
+    datetime. [Bug #24195](https://github.com/rails/rails/issues/24195)
+
+    *Sen Zhang*
```
Comment (arthurnn) [inline guides/CHANGELOG.md]: This changelog should be on activerecord/CHANGELOG.md file.

**Comment by Sen-Zhang** [comment]
@arthurnn Oops, sorry about that. Have fixed.

**Comment by arthurnn** [comment]
thanks

**Pull request merged**